cmake/Lint.cmake

```diff
@@ -52,15 +52,22 @@ function(lint_target TARGET_NAME)
     get_target_property(TARGET_SOURCES ${TARGET_NAME} SOURCES)
     get_target_property(TARGET_SOURCE_DIR ${TARGET_NAME} SOURCE_DIR)
 
-    set(TARGET_OUTPUTS)
-    foreach(SOURCE ${TARGET_SOURCES})
-        lint_file(${TARGET_SOURCE_DIR} ${SOURCE} ${SOURCE}.lint)
-        list(APPEND TARGET_OUTPUTS ${SOURCE}.lint)
-    endforeach()
+    if( NOT Python3_Interpreter_FOUND)
+      find_package(Python3 REQUIRED COMPONENTS Interpreter)
```

Comment:
`REQUIRED` will result in a `FATAL_ERROR` if Python3 isn't found, which doesn't seem to be your intention based on the check after this. You probably want to remove `REQUIRED`.

Reply:
Good point!